processor/transformprocessor/internal/common/metrics.go

```diff
@@ -120,7 +120,7 @@ func (d dataPointStatements) handleNumberDataPoints(ctx context.Context, dps pme
 
 func (d dataPointStatements) handleHistogramDataPoints(ctx context.Context, dps pmetric.HistogramDataPointSlice, metric pmetric.Metric, metrics pmetric.MetricSlice, is pcommon.InstrumentationScope, resource pcommon.Resource) error {
 	for i := 0; i < dps.Len(); i++ {
-		tCtx := ottldatapoint.NewTransformContext(dps.At(i), metric, metrics, is, resource)
+    tCtx := ottldatapoint.NewTransformContext(dps.At(i), metric, metrics, is, resource, pmetric.NewScopeMetrics(), pmetric.NewResourceMetrics())
```

Comment:
We need to pass the actual ScopeMetrics and ResourceMetrics here, otherwise accessing the schema url wont work for histogram datapoints.